res/database/Default/user/scr0llbaer.xml

```diff
@@ -0,0 +1,1586 @@
+				<effect trigger="happen" type="triggernewschoice" time="1,10,15" choose="or" news1="321a8590-eec0-4f1d-8031-6d1f9b443b33" probability1="50" news2="9891cfa6-1cd9-41e4-b179-496da048f7d4" probability2="50" />
+			</effects>
+		</news>
+
+		<news id="321a8590-eec0-4f1d-8031-6d1f9b443b33" type="2" thread_id="51b23dcb-0069-4435-936a-33c6973491c2" creator="9551">
+			<title>
+				<de>Zustand des Wals verschlechtert sich</de>
+				<en>Condition of the whale deteriorates</en>
+			</title>
+			<description>
+				<de>Grienpies: Man müsse umgehend eine Lösung für die Situation finden, sonst überlebe der Wal nicht.</de>
+				<en>Creenpiece: A solution to the situation must be found immediately, otherwise the whale will not survive, Creenpiece said.</en>
+			</description>
+			<data genre="4" price="1" quality="40" fictional="1" />
+			<effects>
+				<effect trigger="happen" type="triggernews" time="1,10,15" news="a33be652-8202-4685-a6ae-1b530f04b3a5" />
+			</effects>
+		</news>
+
+		<news id="a33be652-8202-4685-a6ae-1b530f04b3a5" type="2" thread_id="51b23dcb-0069-4435-936a-33c6973491c2" creator="9551">
+			<title>
+				<de>Wal in Themse ist tot</de>
+				<en>Whale in Thames dead</en>
+			</title>
+			<description>
+				<de>Die genaue Todesursache ist noch unklar, vermutlich war der Wal bereits schwach und konnte nicht genug Nahrung finden.</de>
+				<en>The exact cause of death is still unclear, probably the whale was already weak and could not find enough food.</en>
+			</description>
+			<data genre="4" price="1" quality="70" fictional="1" />
+		</news>
+
+		<news id="9891cfa6-1cd9-41e4-b179-496da048f7d4" type="2" thread_id="51b23dcb-0069-4435-936a-33c6973491c2" creator="9551">
+			<title>
+				<de>Wal in Themse schwimmt zur Flußmündung</de>
+				<en>Whale in Thames swims to river mouth</en>
+			</title>
+			<description>
+				<de>Es sind Fortschritte bei der Rettungsaktion zu verzeichnen. Hoffentlich findet der Wal seinen Weg.</de>
+				<en>Progress is being made in the rescue operation. Hopefully the whale will find its way.</en>
+			</description>
+			<data genre="4" price="1" quality="50" fictional="1" />
+			<effects>
+				<effect trigger="happen" type="triggernews" time="1,10,15" news="5d9a2849-c5f2-4504-912f-84cf5065a9b5" />
+			</effects>
+		</news>
+
+		<news id="5d9a2849-c5f2-4504-912f-84cf5065a9b5" type="2" thread_id="51b23dcb-0069-4435-936a-33c6973491c2" creator="9551">
+			<title>
+				<de>Wal von Themse ins Meer zurückgeschwommen</de>
+				<en>Whale from Thames swam back to sea</en>
+			</title>
+			<description>
+				<de>Na also! Der Wal ist wieder in seinem natürlichen Habitat.</de>
+				<en>There you go! The whale is back in its natural habitat.</en>
+			</description>
+			<data genre="4" price="1" quality="70" fictional="1" />
+		</news>
+	</allnews>
+
+	<allprogrammes>
+		<programme id="f95058fe-1d4e-4d93-83f6-e33ae06ed346" product="2" licence_type="3" fictional="1" creator="9551" comment="Inspired from programme in MadTV">
```

Comment:
I do not think it should be an issue. There is a necessity for "originality levels" things need to fullfill requirements regarding "Urheberrecht".
Movie titles are a bit different (potentially registered trademarks, commercial "usage" etc).

Nonetheless it is always a good idea to add in "comments" what the source of the programme is (so it is easier to find in case there raises an issue about using them)